openedx/features/assessment/helpers.py

```diff
@@ -127,6 +127,9 @@ def autoscore_ora_submission(submission):
     Auto score ORA submission, by Philu bot. Score will appear as awarded by staff (instructor). This function will
     also mark all requirements of submitter to fulfilled, which means all ORA steps will be marked as completed.
```

Comment:
Let's please update it according to the current implementation, as we are not marking all steps as completed.

Reply:
Done